tms-service/tms-service.yml

```diff
@@ -8,7 +8,9 @@ functions:
   frm-tms-service:
     lang: dockerfile
     handler: .
-    image: localhost:5000/frm-tms-service:latest
+    image: localhost:32000/frm_tms_service:latest
```

Comment:
Please rebase or merge the master branch. You'll see deployment and local dev yamls are different. Can you change this file according to the nodejs.template?
(or let's discuss do we need two yaml file? any other solution)
@aaronreynoza @johanfol @JBornman